packages/assets-controllers/src/TokensController.test.ts

```diff
@@ -123,7 +128,7 @@ describe('TokensController', () => {
       onTokenListStateChange,
       config: {
         selectedAddress: defaultSelectedAddress,
-        provider: sinon.stub(),
+        provider,
```

Comment:
Sorry one more thing. We want to avoid the possibility that a test could make a network request, so we shouldn't use a real provider that hits Infura here. We have a version of the provider which intercepts requests before they have a chance to hit the network, so if you put this at the top:

``` typescript
import { FakeProvider } from '../../../tests/fake-provider';
```

and make a new instance of FakeProvider in this `beforeEach`:

``` typescript
const fakeProvider = new fakeProvider();
```

and then pass it here:

``` typescript
provider: fakeProvider,
```

then we can safely protect against making network requests.